shared/js/loadApp.js

```diff
@@ -50,7 +68,8 @@ module.exports = function (callback) {
         .then(loadSource('ace/mode-javascript'))
         .then(loadSource('ace/ext-language_tools'))
         .then(loadSource('droplet/droplet-full'))
-        .then(loadSource('tooltipster/jquery.tooltipster'));
+        .then(loadSource('tooltipster/jquery.tooltipster'))
+        .then(loadExternalSource('https://www.google.com/jsapi'));
```

Comment:
do we have any idea how much code this is loading? it looks like it makes a bunch of other calls to obfuscated members of `google.loader`.